README.md

```diff
@@ -6,5 +6,36 @@ are not redirected but evaluated in the bridge as they require the JVM.
 [nREPL]: https://github.com/clojure/tools.nrepl
 [Arcadia Unity]: https://github.com/arcadia-unity/Arcadia
 
-## How to use
-TBD
+### How to use Cursive with Arcadia
+
+In your Unity project with Arcadia, do:
+
+1. Create a new "regular" looking clojure project that has a deps.edn at its root. For example:
+
+  ```sh
+cd Assets
+mkdir <project-name>
+cd <project-name>
+mkdir src test target
+echo '{:deps {org.clojure/clojure {:mvn/version "1.10.0"}
+              arcadia             {:local/root "../Arcadia"}}}' > deps.edn
+```
+
+2. Open and import the project with Cursive at that root.
```

Comment:
Unclear what this means, do you mean open Cursive? Import Arcadia? Isn't that already done in `deps.edn`?

Reply:
This is outdated and has been reworded.